docs/development/intro.rst

```diff
@@ -147,82 +147,103 @@ certainly the same week), are best.
 Get set up
 ==========
 
-.. warning::
+.. note::
+        :class: sphx-glr-download-link-note
 
-    The rest of this page isn't written yet. It's almost identical to
-    https://ctapipe.readthedocs.io/en/latest/developer-guide/getting-started.html so for
-    now, see there. Also, we shouldn't duplicate content from
-    https://docs.astropy.org/en/latest/development/index.html but link
-    there instead.
+        This section is based on the
+        `instructions <https://ctapipe.readthedocs.io/en/latest/developer-guide/getting-started.html>`_
+        for setting up ``ctapipe``.
 
-The first steps are basically identical to
-https://ctapipe.readthedocs.io/en/latest/developer-guide/getting-started.html (until 
-section *Setting up the development environment*) and
-http://astropy.readthedocs.io/en/latest/development/workflow/get_devel_version.html
-(up to *Create your own private workspace*). The following is a quick summary of
-commands to set up an environment for Gammapy development:
+Firstly, fork the `gammapy repository <https://github.com/gammapy/gammapy>`_ to
+your personal GitHub. If you have not already done so, copy your public SSH key
+(normally under ``$HOME/.ssh/*.pub``) to your GitHub profile by going into your
+GitHub settings and doing **SSH and GPG keys/New SSH key** and selecting **Authentication Key**
+as the key type. If you do not have a SSH key, you may create it following the instructions in
+`this page <https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent>`_.
+
+Go to the location in your machine where you want to code and clone the gammapy
+repository from your GitHub:
+
+.. code-block:: bash
+
+    cd [your-working-path]
+    git clone git@github.com:[your-github-username]/gammapy.git
+
+Create the conda environment:
 
 .. code-block:: bash
 
-    # Fork the gammapy repository on GitHub, https://github.com/gammapy/gammapy
-    cd code # Go somewhere on your machine where you want to code
-    git clone https://github.com/[your-github-username]/gammapy.git
     cd gammapy
     conda env create -f environment-dev.yml
 
-    # To speed up the environment solving you can use mamba instead of conda
-    # mamba env create -f environment-dev.yml
-    conda activate gammapy-dev
-
-    # for conda versions <4.4.0 you may have to execute
-    # 'source activate gammapy-dev' instead
-    git remote add gammapy git@github.com:gammapy/gammapy.git
-    git remote rename origin [your-user-name]
```

Comment:
Maybe one can precise that one can do that, in comment... It is quite used by some